.travis.yml

```diff
@@ -1,11 +1,14 @@
+env:
```

Comment:
I am not familiar with any of this. Is this a starting point for docker software container?